### PR TITLE
Normalize EOL inside comments

### PR DESCRIPTION
**Pull request by leus**
Normalize EOL characters inside comment text.

Also, there are some test fixes for EOL differences under Windows.

See #1200

### CODE REVIEW

**Comment by leus** [comment]
Fixes #713

**Comment by leus** [comment]
Fixes #1047

File: javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java [hunk 1]
```diff
@@ -44,6 +45,15 @@ public PrettyPrinterConfiguration setIndent(String indent) {
         return this;
     }
 
+    public boolean isNormalizeEolInComments() {
```
Comment (ftomassetti) [inline javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java]: maybe in this case the name should be `normalizeEolInComment` or `isEolInCommentsNormalizationEnabled`, not sure

**Review by ftomassetti — APPROVED** [review]
Good stuff!

File: javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java (hunk 1 above)
Comment (leus) [reply javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java]: I think the first one is more consistent

**Comment by matozoid** [comment]
Appveyor is working again! @leus, you are my hero! It was broken for so long...

**Pull request merged**